ci_example_cpp/src/pid.cpp

```diff
@@ -42,8 +42,15 @@ namespace ci_example {
   class Default_pid_factory {
 
   public:
+    /** configs_ contains the PID gains */
```

Comment:
For more compact documentation phrases like "X contains ..." or "This function ..." can typically be dropped.  I think here "The PID gains" would be enough.